project/Merging.scala

```diff
@@ -5,6 +5,7 @@ object Merging {
     case x if x.endsWith("io.netty.versions.properties") => MergeStrategy.discard
     // we target Java 8, which does not use module-info.class. Some dependencies (Jackson) cause assembly problems on module-info
     case x if x.endsWith("module-info.class") => MergeStrategy.discard
+    case x if x.contains("javax/activation") => MergeStrategy.first
```

Comment:
how much would it matter if we replaced this section with:

```
{
 case x => MergeStrategy.first
}
```
https://stackoverflow.com/questions/25144484/sbt-assembly-deduplication-found-error

Reply:
It's pretty rare that these duplicates pop up so personally I prefer dealing with them on a case by case basis rather than handling every case silently. I think it could lead to some tricky debugging if the merge strategy chooses something that the developer isn't expecting

Reply:
+1 to Matt's sentiment!